src/components/referrals/ReferralsComponent.vue

```diff
@@ -72,13 +76,15 @@
                 <button-component
                     :disabled="disableAddTest"
                     wide
+                    tabindex="0"
```

Comment:
button is probably self-indexed. Please delete this and check if is indexed still